### PR TITLE
[MAJOR] Implement business management

### LINKED ISSUE #208: As an admin, I want to have an admin portal to manage user and business
a. Manages details and passwords for businesses that will be using the system. 
b. Option to add and terminate a business

### PR DESCRIPTION
**Pull request by davidthai0387**
closes #208

### CODE REVIEW

**Review by huangs08 — CHANGES_REQUESTED** [review]
Some few comments for typos. The rest looks good!

File: backend/src/main/app.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ import ProjectRoute from './routes/ProjectRoute';
 import CallRoute from './routes/CallRoute';
 import TransactionRoute from './routes/TransactionRoute';
 import EventRoute from './routes/EventRoute';
+import BusinessRoute from './routes/BusinessRoute';
```
Comment (Manraj45) [inline backend/src/main/app.ts:26]: Remove the UserRoute import from this file since in the main the file has been deleted, but the import is still here

**Review by Manraj45 — CHANGES_REQUESTED** [review]

**Comment by Manraj45** [comment]
> Kudos, SonarCloud Quality Gate passed!    Quality Gate passed
> 
> [Bug](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=soen490&pullRequest=239&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=soen490&pullRequest=239&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=soen490&pullRequest=239&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=CODE_SMELL) [1 Code Smell](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=CODE_SMELL)
> 
> [No Coverage information](https://sonarcloud.io/component_measures?id=soen490&pullRequest=239) No Coverage information [](https://sonarcloud.io/component_measures?id=soen490&pullRequest=239&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=soen490&pullRequest=239&metric=new_duplicated_lines_density&view=list)

@nasaku898 @kevin-ghannoum  Should we ignore this code smell?

**Comment by nasaku898** [comment]
> > Kudos, SonarCloud Quality Gate passed!    Quality Gate passed
> > 
> > [Bug](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=soen490&pullRequest=239&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=soen490&pullRequest=239&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=soen490&pullRequest=239&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=CODE_SMELL) [1 Code Smell](https://sonarcloud.io/project/issues?id=soen490&pullRequest=239&resolved=false&types=CODE_SMELL)
> > 
> > [No Coverage information](https://sonarcloud.io/component_measures?id=soen490&pullRequest=239) No Coverage information [](https://sonarcloud.io/component_measures?id=soen490&pullRequest=239&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=soen490&pullRequest=239&metric=new_duplicated_lines_density&view=list)
> 
> @nasaku898 @kevin-ghannoum  Should we ignore this code smell?

Yeah we should ignore this code smell. It's the cleanest way of handling this case and I don't think it's a cyclomatic issue.

**Review by Manraj45 — APPROVED** [review]
Good Job!

**Review by huangs08 — APPROVED** [review]
Changes are made, good to go

**Pull request merged**